build/packaging/rpm/condor.spec

```diff
@@ -1301,23 +1317,30 @@ rm -rf %{buildroot}
 /usr/lib64/python%{python3_version}/site-packages/htcondor2/
 %endif
 
-%files credmon-oauth
+%files credmon-local
 %doc examples/condor_credmon_oauth
 %_sbindir/condor_credmon_oauth
 %_sbindir/scitokens_credential_producer
-%_var/www/wsgi-scripts/condor_credmon_oauth
 %_libexecdir/condor/credmon
 %_var/lib/condor/oauth_credentials/README.credentials
 %config(noreplace) %_sysconfdir/condor/config.d/40-oauth-credmon.conf
```

Comment:
Should this me 40-local-credmon.conf? If so, we may need a rename elsewhere. I see that this file name documented below. Still, it might be less confusing to change the file name.

Reply:
I would like to put the local issuer config in its own file, but I'm not sure how to do that without either:
1. Adding a new local issuer config file, which might confuse admins who have already configured the local issuer in a different config file, or
2. Moving the existing `40-oauth-credmon.conf` to `40-local-credmon.conf` and adding a new `40-oauth-credmon.conf`, which would confuse admins who have already configured the OAuth credmon.

For the case in (1), I'm also not sure if having `use feature: oauth` twice in the config would cause problems, I suppose I could test it.